pkg/util/operator_config.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2018 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     https://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package util
+
+type OperatorConfig struct {
+	Namespace               string
+	SubmissionRunnerThreads int
+	UnifiedSparkImage       string
```

Comment:
You can put all metrics-related field into a `MetricConfig` and embed `MetricConfig` inside the `OperatorConfig`.